ts/Series/Pie/PieDataLabel.ts

```diff
@@ -656,6 +656,9 @@ namespace ColumnDataLabel {
                     relativeLength(options.innerSize || 0, newSize),
                     newSize
                 );
+                options.thickness ? // #6647
+                newSize - options.thickness * 2
+                :
```

Comment:
The use of a ternary operator here is wrong. This line `newSize - options.thickness * 2` isn't doing anything.

Reply:
Yes, I missed a line `relativeLength(options.innerSize || 0, newSize), newSize);`

Reply:
- [ ] Is the comment here needed or useful? To get the related issue for a given like we could use git blame.

- [ ] The indentation is off and the below looks better (just the format, not the content):
```
                options.thickness ? // #6647
                    newSize - options.thickness * 2 :
                    this.translate(center);
```